test/nightwatch_tests/helpers/helpers.js

```diff
@@ -25,23 +26,31 @@ module.exports = {
 
     console.log("nano setting up database");
     // clean up the database we created previously
-    nano.db.destroy(database, function (err, body, header) {
-      if (err) {
-        if (err.message != 'Database does not exist.' && err.message != 'missing') {
-          console.log('Error in setting up ' + database, err.message);
-        }
-      }
-      // create a new database
-      nano.db.create(database, function (err, body, header) {
-        if (err) {
-          console.log('Error in setting up ' + database, err.message);
-        }
+    async.parallel([
+      function (cb) {
+        nano.db.destroy(database, function (err, body, header) {
+          if (err && err.message !== 'Database does not exist.' && err.message !== 'missing') {
+            console.log('Error in setting up ' + database, err.message);
+          }
+          // create a new database
+          nano.db.create(database, function (err, body, header) {
+            if (err) {
+              console.log('Error in setting up ' + database, err.message);
+            }
+            cb();
+          });
+        });
+      },
+      function (cb) {
         nano.db.create('_replicator', function (err, body, header) {
```

Comment:
it creates the replicator db in parallel to the fauxton-seleniumtest db, shaving off a few second each time (https://github.com/caolan/async#parallel) instead of running them chained in a synchronously manner